### PR TITLE
Fix Misspellings across pages and posts

### PR DESCRIPTION
**Pull request by elisoncrum**
## Summary
Fixes several instances of spelling or spacing across _pages, _posts, or assets.

### CODE REVIEW

File: _posts/2020-12-16-Hackathon-4-Success.md [hunk 1]
```diff
@@ -3,6 +3,6 @@ Our latest Hackathon was as a huge success!  We added many new features to the M
 After the hackathon, we hosted our first guest speaker, Miles Tiglao (USMC, ret.)!  Checkout the recording of his talk at the links below:
 
 > For our non-CAC members, [Zoom Recording](https://us02web.zoom.us/rec/share/8qntZYHsw4H1xkvCFS6YfwuYNu9eweyhi22OUYHwsQJwbFywtCqadBkYzQ_0nsFY.Szee4QhLQhLRYIGP)
-* Passcocde: Z?E$C28k
+* Passcode: Z?E$C28k
```
Comment (SemperRuminare) [inline _posts/2020-12-16-Hackathon-4-Success.md:6]: Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears your code contains a hardcoded password. Hardcoded secrets or credentials can allow attackers to bypass authentication methods and perform malicious actions. We recommend revoking access to resources using this credential and storing future credentials in a management service such as [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/).

[Learn more about the use of hardcoded credentials](https://cwe.mitre.org/data/definitions/798.html)

File: _pages/events.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ November 20, 2020 - [Brief](https://www.youtube.com/watch?v=XybGruUNc7s&t=5s) to
 Recap of our guest speaker, Miles Tiglao:
 
 > For our non-CAC members, [Zoom Recording](https://us02web.zoom.us/rec/share/8qntZYHsw4H1xkvCFS6YfwuYNu9eweyhi22OUYHwsQJwbFywtCqadBkYzQ_0nsFY.Szee4QhLQhLRYIGP) 
-* Passcocde: Z?E$C28k
+* Passcode: Z?E$C28k
```
Comment (SemperRuminare) [inline _pages/events.md:37]: Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears your code contains a hardcoded password. Hardcoded secrets or credentials can allow attackers to bypass authentication methods and perform malicious actions. We recommend revoking access to resources using this credential and storing future credentials in a management service such as [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/).

[Learn more about the use of hardcoded credentials](https://cwe.mitre.org/data/definitions/798.html)

File: _posts/2020-12-16-Hackathon-4-Success.md (hunk 1 above)
Comment (elisoncrum) [reply _posts/2020-12-16-Hackathon-4-Success.md:6]: Good bot. These are meant to be hard coded, information is public and meant to distribute Zoom recording.

File: _pages/events.md (hunk 2 above)
Comment (elisoncrum) [reply _pages/events.md:37]: Good bot. These are meant to be hard coded, information is public and meant to distribute Zoom recording.

**Comment by SemperRuminare** [comment]
Thanks looks good

**Pull request merged**